hieradata/clients/pkg.yaml

```diff
@@ -0,0 +1,3 @@
+---
+apache::mod::event::serverlimit: '30' #default 25
+apache::mod::event::maxrequestworkers: '750' #ServerLimit * 25
```

Comment:
- Can you add a comment linking to the apache doc. with the formula?
- Where does the 25 value come from?